packages/vaadin-time-picker/src/vaadin-time-picker.js

```diff
@@ -85,15 +85,13 @@ class TimePickerElement extends ElementMixin(ControlStateMixin(ThemableMixin(Pol
       </style>
       <vaadin-combo-box-light
         allow-custom-value=""
-        item-label-path="value"
         filtered-items="[[__dropdownItems]]"
         disabled="[[disabled]]"
         readonly="[[readonly]]"
         auto-open-disabled="[[autoOpenDisabled]]"
         dir="ltr"
         theme$="[[theme]]"
       >
-        <template> [[item.label]] </template>
```

Comment:
This template is not necessary here as by default the combo-box already renders `item.label` without templates. In fact, this template is not used by anyone.